docs/config/README.md

```diff
@@ -989,6 +990,33 @@ The module will be shown if any of the following conditions are met:
 symbol = "⚙️ "
 ```
 
+## Terraform
+
+The `terraform` module shows the currently selected terraform workspace and version.
+By default the terraform version is not shown, since this is slow on current versions of terraform when a lot of plugins are in use.
+The module will be shown if any of the following conditions are met:
+
+- The current directory contains a `.terraform` folder
+- Current directory contains a file with the `.tf` extension
+
+### Options
+
+| Variable       | Default       | Description                                                 |
+| -------------- | ------------- | ----------------------------------------------------------- |
+| `symbol`       | `"💠 "`       | The symbol used before displaying the terraform workspace.  |
+| `style`        | `"bold 105"`  | The style for the module.                                   |
+| `disabled`     | `false`       | Disables the `terraform` module.                            |
+| `show_version` | `false`       | Shows the terraform version. Very slow on large workspaces. |
```

Comment:
Nitpick: To keep the "Options" tables consistent, `style` and `disabled` are kept as the last two rows. Could we please move `show_version` to be above them? 😊